yaksh/test_views.py

```diff
@@ -206,6 +207,30 @@ def test_edit_profile_get(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'yaksh/editprofile.html')
 
+    def test_edit_profile_get_for_user_without_profile(self):
+        """
+        If no profile exists a blank profile form will be displayed
+        """
+        self.client.login(
+            username=self.user1.username,
+            password=self.user1_plaintext_pass
+        )
+        response = self.client.get(reverse('yaksh:edit_profile'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'yaksh/editprofile.html')
+
+    def test_edit_profile_get_for_user_with_profile(self):
+        """
+        If profile exists a editprofile.html template will be rendered
+        """
+        self.client.login(
+            username=self.user2.username,
+            password=self.user2_plaintext_pass
+        )
+        response = self.client.get(reverse('yaksh:edit_profile'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'yaksh/editprofile.html')
+
```

Comment:
Done, added tests for all 3 functions where the has_profile decorator is being used